client/components/web-preview/style.scss

```diff
@@ -22,49 +40,41 @@
 }
 
 .web-preview__content {
-	box-shadow: 0 0 0 1px transparentize( lighten( $gray-dark, 20% ), .5 ),
-		0 1px 2px lighten( $gray-dark, 30% );
+	box-shadow: 0 0 0 1px rgba( $gray, 0.5 );
+	background: $gray-light;
 	border-radius: 4px 4px 0 0;
 	position: absolute;
-		top: 47px;
+		top: 24px;
 		left: 0;
 		right: 0;
 		bottom: 0;
 	margin: 0 auto;
-	transform: scale( 0.96 );
-	transition: 0.1s transform cubic-bezier( 0.25, 0.5, 0.5, 0.9 );
-	background: lighten( $gray, 30% );
+	opacity: 0;
+	transform: translateY( 80vh );
+	transition: transform 0.2s ease-out,
+		opacity 0.1s ease-in-out,
+		max-width 0.2s ease-out;
 
 	.is-computer & {
-		width: 100%;
+		max-width: 1040px;
```

Comment:
> I quite liked that it left plenty of space on the sides on a 1440px screen. It made it more apparent that it's something on top of the Calypso UI I guess.

That was the intention with making the max-width 1040px — making it full width but still having the space above the preview felt really strange and lost a lot of the metaphor of the preview "floating on top" of Calypso. Perhaps a better solution is to add a padding/margin so there's always at least 24px (or something) on the sides for wider viewports. Thoughts @mtias @lsinger @folletto ?

What are these "tools" you speak of?